CONTRIBUTING.md

```diff
@@ -41,313 +27,228 @@ process documented here.
+```
 
-  Click "fork" to create "your copy" of the repository you want to change on
-  https://github.com/kata-containers/community. This is an exact copy of the
-  upstream repository located at
-  `https://github.com/${your-github-username}/community` on the GitHub server. You
-  can make changes on this copy as we move through the example.
+The issue is automatically closed by GitHub when the
+[commit message](https://help.github.com/articles/closing-issues-via-commit-messages/) is parsed.
 
-- Prepare your environment
+## GitHub workflow
 
-  > **Note:**: Most of the
-  > [Kata Containers repositories](https://github.com/kata-containers)
-  > contain code written in the
-  > [Go language (golang)](https://golang.org/). Go requires all code to be put
-  > inside the directory specified by the `$GOPATH` variable. Since this example PR
-  > is not using golang you do not need to
-  > [install golang](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#requirements-to-build-individual-components).
-  > However it still makes sense to put the code in the standard location.
+Kata Containers employs certain augmentations to a 
+[standard GitHub workflow](https://guides.github.com/introduction/flow/). 
+In this section, we explain these augmentations in more detail. Follow these guidelines when contributing to Kata Containers repositories, except where noted below.
 
-  ```sh
-  $ export GOPATH=${GOPATH:-$HOME/go}
-  $ mkdir -p "$GOPATH"
-  ```
+* Complete the [GitHub basic setup](#github-basic-setup) above before continuing.
 
-  > **Note:** The code above is safe to run whether you have golang installed
-  > or not.
-  > For further details on golang, refer to the
-  > [requirements section of the Kata Developer Guide](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#requirements-to-build-individual-components).
+* Ensure each PR only covers one topic. If you mix up different items in
+  your patches or PR, they will likely need to be reworked.
 
+* Follow a [topic branch method](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-branches) for development.
 
-- [Clone the upstream repository](https://help.github.com/articles/cloning-a-repository):
+* Follow carefully the [patch format](#patch-format) for PRs.
 
-  "Cloning" is a term that means to create a *local copy* of a Git repository
-  that generally lives on a remote server.
+* Apply the appropriate GitHub labels to your PR. This
+  is particularly relevant to maintain [Stable branch backports](#stable-branch-backports). See also [GitHub labels and keywords that block PRs](#github-labels-and-keywords-that-block-prs)
 
-  ```sh
-  $ dir="$GOPATH/src/github.com/kata-containers"
-  $ mkdir -p "$dir"
-  $ cd "$dir"
-  $ git clone https://github.com/kata-containers/community
-  $ cd community
-  ```
+  >*Note:* External contributors should use keywords, explained in the link 
```

Comment:
Nit: For consistency with our other docs, this should strictly be:

```
> **Note:** External ...
```

So that it renders as:

> **Note:** External ...

Same comment for some of the other new "notes" below...

See https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#notes.